history-of-the-quakers-v3/original/16-book-5-chapter-1.adoc

```diff
@@ -62,8 +65,13 @@ Accordingly, about a month after this king`'s accession to the throne, George Wh
 Gilbert Latey and Alexander Parker waited upon him with the following address,
 and case of their friends:
 
+[.embedded-content-document.address]
```

Comment:
Another funky letter, and I can't remember how we've done them!!! I'm sorry!!!

Reply:
yup, funky indeed.  I think i like how you did it here. 👍